src/ModelManager.ts

```diff
@@ -54,6 +54,14 @@ export interface ModelManagerConfiguration {
     path?: string;
 }
 
+interface ModelPaths {
+    rootModelURL?: string;
+    rootModelPath?: string;
+    currentPathname?: string | null;
+    sanitizedCurrentPathname?: string;
+    metaPropertyModelUrl?: string;
+}
```

Comment:
these variable names could be optimized, WDYT? If we want to restructure and keep a reference to the internal variables, we may also want to rename the fields of that interface.

* Why do we have `rootModelURL` and `rootModelPath`? That is confusing.
* Why do we keep a reference to `currentPathname` and `sanitizedCurrentPathname`? Also, I suppose that sanitization is internal business.
* Do we need to keep multiple variation of the ModelURL in memory? Also, `rootModelURL`, `metaPropertyModelUrl` have 2 different naming conventions for the URL.

Reply:
@pfauchere 
- `rootModelPath` is the sanitized `rootModelURL`.It's being used in multiple places so stored it to avoid duplicate calls to `.sanitize()`.Is this too confusing? WDYT?
- Again, one is the sanitized version of the other, but not used as much as `rootModelPath`.So will clean this up.
- `metaPropertyModelUrl` is also used in different contexts, hence why it was stored. Will fix the naming convention in the meantime, thanks!

Reply:
> rootModelPath is the sanitized rootModelURL.It's being used in multiple places so stored it to avoid duplicate calls to .sanitize().Is this too confusing? WDYT?

I am all for optimization and avoid to call sanitize more than once. But then do we need to keep a reference to the unsanitized version of the URL?

Reply:
> But then do we need to keep a reference to the unsanitized version of the URL?

In the existing implementation, that is used quite frequently as well.